app/src/main/java/com/example/androidDeviceDetails/models/location/LocationItemViewHolder.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.androidDeviceDetails.models.location
+
+import com.example.androidDeviceDetails.adapters.LocationAdapter
+
```

Comment:
- This is just a data class not view holder rename accordingly.

Reply:
done